packages/components/Menu/src/MenuList/MenuList.tsx

```diff
@@ -44,23 +43,13 @@ export const MenuList = compose<MenuListType>({
   slots: {
     root: MenuStack,
     scrollView: ScrollView,
-    ...(Platform.OS === 'macos' && { focusZone: FocusZone }),
+    focusZone: Platform.OS !== 'android' ? FocusZone : React.Fragment,
```

Comment:
Might be more accurate to do like a `supportsFocusZone = ['macos', 'win32'] and check those two so that other platforms like windows and iOS don't immediately Redbox.